src/helpers.js

```diff
@@ -11,19 +11,26 @@ function transposeDistance(transposeKey, songKey) {
 }
 
 /* eslint import/prefer-default-export: 0 */
-export function renderChord(chord, lineKey, songKey, transposeKey) {
-  const distance = presence(transposeKey) ? transposeDistance(transposeKey, songKey) : 0;
+export function renderChord(chord, lineKey, songKey, transposeKey, capo) {
   let chordObj = Chord.parse(chord);
 
   if (!chordObj) {
     return chord;
   }
 
-  if (presence(transposeKey) && presence(songKey)) {
-    chordObj = chordObj.transpose(distance).useModifier(transposeKey.modifier);
+  let transpose = -1 * (capo || 0);
```

Comment:
Can you explain what this does? You pass the capo value into the renderChord function and then it will multiply -1 by the capo value?

Reply:
It subtracts the capo level from the transpose level. So for capo 3 it will transpose the chords down by 3 semitones. It also take in account the transpose level, so capo 3 combined with transpose 1 means effectively transposing by -2.

Reply:
Can you show me an example of where or how that would be used? When I saw that code initially It made me think that it really was going to function like this:

```
{key: C}
// played key : C
// concert key : C
Let it [Am]be, let it [C/G]be, let it [F]be, let it [C]be

// ... setCapo(3)

{key: C}
{capo: 3}
// played key : A
// concert key : C
Let it [F#m]be, let it [A/E]be, let it [D]be, let it [A]be
```

Where the capo has an impact on the 'transposing'. 

Or take this example:
```
  it('handles capo with changing key', () => {
    const chordpro = `
{key: C}
{capo: 3}
Let it [Am]be, let it [C/G]be, let it [F]be, let it [C]be`.substring(1);

    const changedSheet = `
{key: A}
{capo: 3}
Let it [F#m]be, let it [A/E]be, let it [D]be, let it [A]be`.substring(1);

    const song = new ChordProParser().parse(chordpro);
    const updatedSong = song.setCapo(3);
    const transposedSong = updatedSong.setKey('A');

    expect(new ChordProFormatter().format(transposedSong)).toEqual(changedSheet);
  });
```

The above test passes, but when the song is transposed to 'A' the capo doesn't have any effect on it. 

When I see that the renderChord function accepts a capo that affects the transposing I would have anticipated that the capo is having some kind of impact on the chordpro.

Reply:
I see it now. When it's actually formatted it uses the capo to transpose it.